source_code/caches/.gitignore

```diff
@@ -0,0 +1,43 @@
+work
```

Comment:
Merge this with the top-level .gitignore and remove. No reason to have this in more than one place

Reply:
removed